tests/test_meshtastic_utils_coverage.py

```
@@ -32,9 +32,36 @@
+        A Future already completed with the value `None`.
+    """
+    close = getattr(coro, "close", None)
+    if callable(close):
+        close()
+    done_future: Future[None] = Future()
+    done_future.set_result(None)
+    return done_future
+
+
 @pytest.fixture(autouse=True)
 def reset_meshtastic_state(reset_meshtastic_globals):
```

Ruff (0.15.10)

[warning] Unused function argument: `reset_meshtastic_globals`

(ARG001)